packages/apptive_grid_form/lib/src/apptive_grid_form.dart

```diff
@@ -492,10 +494,18 @@ class ApptiveGridFormDataState extends State<ApptiveGridFormData> {
           ),
         ),
         Text(
-          localization.sendSuccess,
+          _formData?.successTitle ?? localization.sendSuccess,
           textAlign: TextAlign.center,
           style: Theme.of(context).textTheme.headline4,
         ),
+        if (_formData?.successMessage != null) ...[
+          const SizedBox(height: 4),
+          Text(
+            _formData!.successMessage!,
+            textAlign: TextAlign.center,
+            style: Theme.of(context).textTheme.bodyMedium,
+          ),
+        ],
         Center(
           child: TextButton(
```

Comment:
Would it make sense to allow customisation of this text similar to how the package allows for a custom send button text?

Reply:
Seems like something we could do as a nice to have feature, but maybe not in this PR. I'll make a ticket for it

Reply:
https://2denker.atlassian.net/browse/AC-2325